library/tiqr/Tiqr/StateStorage/Abstract.php

```diff
@@ -41,27 +41,6 @@ abstract class Tiqr_StateStorage_Abstract
      */
     protected $logger;
 
-    /**
-     * Store a value with a certain key in the statestorage.
-     * @param String $key The key identifying the data
-     * @param mixed $value The data to store in state storage
-     * @param int $expire The expiration (in seconds) of the data
-     */
-    public abstract function setValue($key, $value, $expire=0);
-    
-    /**
-     * Remove a value from the state storage
-     * @param String $key The key identifying the data to be removed.
-     */
-    public abstract function unsetValue($key);
-    
-    /**
-     * Retrieve the data for a certain key.
-     * @param String $key The key identifying the data to be retrieved.
-     * @return mixed The data associated with the key
-     */
-    public abstract function getValue($key);
-    
```

Comment:
Extracted this to an interface, and implemented that interface on the File implementation. Left the abstract implementation in place for the siblings that still rely on it.